### PR TITLE
Spanish translation of acceptable files in templates

### PR DESCRIPTION
**Pull request by jmvbxx**
As per the guidelines, only files which are already translated into French can be translated into another language. There are currently two templates which have French translations. 

- overview.html
- sponsors.html

### CODE REVIEW

**Comment by jmvbxx** [comment]
Thanks for your input! And for accepting my previous PR (my first ever to an open solution) :smile: 

I'll finish the remaining templates files mentioned above. Would you like the `locales` file translation included in this file or would you prefer a new branch?

**Comment by stanislas-m** [comment]
> I'll finish the remaining templates files mentioned above. Would you like the locales file translation included in this file or would you prefer a new branch?

When I work on the French version, I translate each page in a different PR. This way, it's easier to review and we can merge things while still working on the other pages. :)

**Comment by jmvbxx** [comment]
@stanislas-m Sounds good. I'll stop pushing changes to this PR. Please consider it complete and ready for review. From now I'll create a separate branch and PR for each translated page.

File: templates/docs/db/_sql.es.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Migraciones de SQL
+
+Si no desea usar Fizz, o si tiene una consulta complicada que desea ejecutar, puede usar SQL.
+
+Para generar una nueva migración ** vacía **, use el siguiente comando:
```
Comment (paganotoni) [inline templates/docs/db/_sql.es.md]: Creo que aquí la palabra "vacía" debería estar mas cerca de los asteriscos

File: templates/docs/db/buffalo-integration.es.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    return app
+}
+```
+
+`poptx.PopTransaction(models.DB)` utiliza la conexión a la base de datos configurada para crear un nuevo middleware `PopTransaction`. Este middleware hace lo siguiente:
+
+* Registre la duración total gastada durante la solicitud de llamadas a la base de datos.
+* Ajustar ** cada solicitud HTTP ** en una transacción de base de datos.
```
Comment (paganotoni) [inline templates/docs/db/buffalo-integration.es.md]: Lo mismo acá, no deberían haber espacios entre la palabra y los asteriscos

File: templates/docs/db/configuration.es.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+<% seoDescription("¿Cómo configurar mi base de datos con Pop?") %>
+<% seoKeywords(["buffalo", "go", "golang", "base de datos", "ORM", "pop", "configuración"]) %>
+
+<%= h1("Configuración de la base de datos") %>
+
+La configuración Pop se gestiona mediante un archivo `database.yml`, ubicado en la raíz de su proyecto. Este archivo se genera para usted si utiliza Buffalo &#8211; si elige usar Pop &#8211; y contiene una configuración básica para la base de datos que seleccionó en la generación con el _flag_ `--db-type`. PostgreSQL se considera como el predeterminado.
```
Comment (paganotoni) [inline templates/docs/db/configuration.es.md]: podríamos decir "PostgreSQL se usa como base de datos predeterminada"?

**Comment by paganotoni** [comment]
@jmvbxx It looks great! added some comments for your consideration.

File: templates/docs/db/_fizz.es.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+### Migraciones Fizz
+
+El comando `soda` generará migraciones de SQL (tanto arriba como abajo) para usted.
+
+```bash
+$ soda generate fizz name_of_migration
+```
+
+Al ejecutar este comando, se generarán los siguientes archivos **vacios**:
```
[the remark this answers is not in the document]
Comment (jmvbxx) [reply templates/docs/db/_fizz.es.md]: Corrected.

File: templates/docs/db/_sql.es.md (hunk 1 above)
Comment (jmvbxx) [reply templates/docs/db/_sql.es.md]: Fixed spacing

File: templates/docs/db/buffalo-integration.es.md (hunk 2 above)
Comment (jmvbxx) [reply templates/docs/db/buffalo-integration.es.md]: Fixed all the of text bolding

File: templates/docs/db/configuration.es.md (hunk 3 above)
Comment (jmvbxx) [reply templates/docs/db/configuration.es.md]: Updated

**Comment by jmvbxx** [comment]
Thank you @stanislas-m .. Fixing with a new PR now.

**Pull request merged**